src/CftfBundle/Resources/views/DocTree/view.html.twig

```diff
@@ -5,510 +5,625 @@
+                        <button type="button" class="treeCheckboxMenuBtn btn btn-default dropdown-toggle"
+                                data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" style="display:none">Action <span
+                                    class="caret"></span></button>
                         <ul class="dropdown-menu">
                             <li><a class="treeCheckboxMenuItem" data-cmd="delete">Delete</a></li>
                             <!-- <li><a class="treeCheckboxMenuItem" data-cmd="edit">Edit</a></li> -->
                             <li><a class="treeCheckboxMenuItem" data-cmd="makeFolders">Make Items Parents</a></li>
                             <li><a class="treeCheckboxMenuItem" data-cmd="hideCheckboxes">Hide Checkboxes</a></li>
                         </ul>
                     </span>
-            {% endif %}
-                    <span class="filterClear glyphicon glyphicon-remove-circle"></span>
-                    <input type="text" class="form-control treeFilter" id="filterOnTree" placeholder="Search...">
-            {% if editorRights and isDraft %}
+                        {% endif %}
+                        <span class="filterClear glyphicon glyphicon-remove-circle"></span>
+                        <input type="text" class="form-control treeFilter" id="filterOnTree" placeholder="Search...">
+                        {% if editorRights  and isDraft %}
+                    </div>
+                    {% endif %}
                 </div>
-            {% endif %}
-        </div>
-
-        <section id="tree1Section">
-            <div id="viewmode_tree1" class="treeDiv"></div>
-        </section>
 
-        <div id="noItemsInstructions" class="bg-info instructions" style="display:none">This document does not currently contain any statements/items.
-            <ul>
-            {% if editorRights and isDraft %}
-                <li>To <b>create statements/items</b> click the "Add New Child Item" button at right.</li>
-                <li>To <b>create "crosswalk" associations</b> between two different documents, start by clicking "Change Document" above, then choose a different document to view here.</li>
-            {% endif %}
-                <li>To <b>view existing crosswalk associations</b>, click the "Association View" button above.</li>
-            </ul>
+                <section id="tree1Section">
+                    <div id="viewmode_tree1" class="treeDiv"></div>
+                </section>
+
+                <div id="noItemsInstructions" class="bg-info instructions" style="display:none">This document does not
+                    currently contain any statements/items.
+                    <ul>
+                        {% if editorRights and isDraft %}
+                            <li>To <b>create statements/items</b> click the â€œAdd New Child Itemâ€� button at right.
```

Comment:
Lines 117 through 123 have the quotes munged to be different characters.

Around `Add New Child Item`, `crosswalk`, `Change Document` and `Association View`